MAINTAINERS.md

```diff
@@ -1,98 +1,73 @@
 
-This file lists how the Habitat project is maintained and the humans and
-companies who make up "The Habitat Maintainers."
+This file lists how the Builder project is maintained and the humans and
+companies who make up "The Builder Maintainers."
 
 When making changes to the system, this file tells you who needs to review your
 patch - you need approval of at least one maintainer for the relevant subsystems
-to provide a :+1: (or `+r`) on your pull request. Additionally, you need to not
+to provide a Github Approval on your pull request. Additionally, you need to not
 receive a veto from a Lieutenant or the Project Lead.
 
 Check out the [Maintenance Policy](maintenance-policy.md) for details on the
 process, how to become a maintainer, lieutenant, or the project lead.
 
 ---
 
-# The Habitat Maintainers
+# The Builder Maintainers
 
-These are the humans who maintain Habitat.  These people have permission to
+These are the humans who maintain Builder. These people have permission to
 merge code changes to the various components and other rights and
 responsibilities as detailed in the [Maintenance Policy](maintenance-policy.md).
 
 # Project Lead
 
-* [Adam Jacob](https://github.com/adamhjk)
+* [Salim Alam](https://github.com/chefsalim)
 
 ## Components
 
-The various components that make up Habitat. Each component must have one
+Below are key components that make up Builder. Each component must have one
 Lieutenant and may have many maintainers.
 
-### Core Habitat
-
-See https://github.com/habitat-sh/habitat/blob/master/MAINTAINERS.md
-
-### Habitat Builder Web Application
-
-The web application for the Habitat builder and depot.
+### Backend and API
 
 #### Lieutenant
 
-* [Chris Nunciato](https://github.com/cnunciato)
+* [Ian Henry](https://github.com/eeyun)
 
 #### Maintainers
 
 * [Fletcher Nichol](https://github.com/fnichol)
-* [Jamie Winsor](https://github.com/reset)
+* [Josh Black](https://github.com/raskchanky)
 * [Matt Wrock](https://github.com/mwrock)
-* [Salim Alam](https://github.com/chefsalim)
-* [Nell Shamrell-Harrington](https://github.com/nellshamrell)
-* [Tasha Drew](https://github.com/tashimi)
 * [Matthew Peck](https://github.com/mpeck)
+* [Scott Macfarlane](https://github.com/smacfarlane)
+
+### Web Application
 
-### Habitat Builder
+The web application for the Habitat builder and depot.
 
 #### Lieutenant
 
-* [Jamie Winsor](https://github.com/reset)
+* TBD
 
 #### Maintainers
 
-* [Adam Jacob](https://github.com/adamhjk)
-* [Fletcher Nichol](https://github.com/fnichol)
-* [Brian Scott](https://github.com/bscott)
-* [Steven Murawski](https://github.com/smurawski)
-* [Ian Henry](https://github.com/eeyun)
-* [Matt Wrock](https://github.com/mwrock)
-* [Salim Alam](https://github.com/chefsalim)
-* [Nell Shamrell-Harrington](https://github.com/nellshamrell)
-* [Jon Bauman](https://github.com/baumanj)
-
-### Documentation
+* [Matthew Peck](https://github.com/mpeck)
+* [Shadae](https://github.com/shadae)
```

Comment:
Is there a reason Shadae doesn't have a last name?

Reply:
Will confirm